### ISSUE TITLE
Let us choose rule name for azure sql firewall rule

### ISSUE THREAD

**Issue by mcenzato** [microsoft/azuredatastudio#13273]
Like in SSMS, it will be much useful to allow the user to choose the name for azure sql firewall rule automatically created at logon phase if IP is not already present in firewall exceptions.
![firewallRule](https://user-images.githubusercontent.com/8800578/98350626-e69a2680-201b-11eb-87a7-086565ceca06.png)

**Comment by longzheng** [comment]
+1 on this request to make it easier to audit who added the rules if there are multiple users.

Alternatively would also be reasonable if the auto-generated firewall rule had a name that included the signed in Azure account name.

For example the current rules are named
```
ClientIPAddress_2022-01-19_02:32:03
```
it could be the name and email of the user, with the timestamp
```
Doe, John - john.doe@email.com_2022-01-19_02:32:03
```

**Comment by longzheng** [comment]
Relates to https://github.com/microsoft/azuredatastudio/issues/13273

**Comment by cheenamalhotra** [comment]
Closing as duplicate of #3182 - will be addressed with PR #21430
